Sourcery/Sourcery.swift

```diff
@@ -38,9 +38,9 @@ public class Sourcery {
     // content annotated with file annotations per file path to write it to
     fileprivate var fileAnnotatedContent: [Path: [String]] = [:]
 
-    private (set) var numberOfFilesThatHadToBeParsed: Int32 = 0
+    private (set) var numberOfFilesThatHadToBeParsed = 0
     func incrementFileParsedCount() {
-        OSAtomicIncrement32(&numberOfFilesThatHadToBeParsed)
```

Comment:
When I ran TSAN on the codebase this increment was causing some violations to be printed, and they went away when I switched to a `DispatchQueue.sync` as a barrier. However even with that change I still encountered the crashes.

Reply:
oh that's really interesting, would not expect that for sure